### ISSUE TITLE
Some Suggestions

### ISSUE THREAD

**Issue by yoge67855** [sandstorm-io/sandstorm#3509]
1. Add these apps:
 searX
 PrivateBin
 Mastodon
 Friendica
 PixelFed
 Tildes
 Nextcloud
 Mumble
 WriteFreely
 HedgeDoc
 XMPP
 Invidious
 Bibliogram
 Nitter
 CyberChef
 CryptPad
 PeerTube
 IRC
 GitEa
 Discourse
 StatusPage
Element
Jitsi Meet
2. Add themes and dark mode
3. update rocket chat

**Comment by ocdtrekkie** [comment]
I would also really like dark mode (#3229) and another issue exists for themes (#2893).

The Rocket.Chat team chose to remove support for Sandstorm from their project because the port wasn't being well-maintained (https://github.com/RocketChat/Rocket.Chat/pull/13773). They would probably be open to supporting Sandstorm again, however someone would have to take on the project of re-adding Sandstorm support, which would not be a small project.

We'd definitely like to see some of the apps you listed on Sandstorm, but there's a lot of work involved that various people would need to take on. You listed several ActivityPub-federated apps, for example (Mastodon, PixelFed, PeerTube), which would really be blocked on Sandstorm having a method to support ActivityPub federation. Similarly, IRC and Element would need support for Matrix federation, which some members of our community have discussed at length as well. I believe something like Jitsi Meet (which we love here) would require some work on allowing grains to request browser features like the camera and microphone, which currently they cannot do.

For what it's worth, someone did try to package CyberChef, and it is in the experimental app market at https://apps.sandstorm.io/app/0v5zvq5faz979cq7mj27pete5s6du0ph780dh75eaax08v6vumr0?experimental=true though it does not retain data, because CyberChef is a client-side app which attempts to use localStorage in the browser, which Sandstorm does not support (#91).

**Comment by yoge67855** [comment]
> I would also really like dark mode (#3229) and another issue exists for themes (#2893).
> 
> The Rocket.Chat team chose to remove support for Sandstorm from their project because the port wasn't being well-maintained ([RocketChat/Rocket.Chat#13773](https://github.com/RocketChat/Rocket.Chat/pull/13773)). They would probably be open to supporting Sandstorm again, however someone would have to take on the project of re-adding Sandstorm support, which would not be a small project.
> 
> We'd definitely like to see some of the apps you listed on Sandstorm, but there's a lot of work involved that various people would need to take on. You listed several ActivityPub-federated apps, for example (Mastodon, PixelFed, PeerTube), which would really be blocked on Sandstorm having a method to support ActivityPub federation. Similarly, IRC and Element would need support for Matrix federation, which some members of our community have discussed at length as well. I believe something like Jitsi Meet (which we love here) would require some work on allowing grains to request browser features like the camera and microphone, which currently they cannot do.
> 
> For what it's worth, someone did try to package CyberChef, and it is in the experimental app market at https://apps.sandstorm.io/app/0v5zvq5faz979cq7mj27pete5s6du0ph780dh75eaax08v6vumr0?experimental=true though it does not retain data, because CyberChef is a client-side app which attempts to use localStorage in the browser, which Sandstorm does not support (#91).

OK

**Comment by ocdtrekkie** [comment]
Going to close this because this is not really an actionable issue for us right now. We'd definitely like a lot of the same apps to come to Sandstorm but someone would need to work on producing packages for each one of them.
